mobile/library/common/http/client.cc

```diff
@@ -442,34 +442,34 @@ void Client::DirectStreamCallbacks::latchError() {
   if (info.responseCode().has_value()) {
     error_->error_code_ = Bridge::Utility::errorCodeFromLocalStatus(
         static_cast<Http::Code>(info.responseCode().value()));
-    error_msg_details.push_back(absl::StrCat("RESPONSE_CODE: ", info.responseCode().value()));
+    error_msg_details.push_back(absl::StrCat("envoy_rc: ", info.responseCode().value()));
   } else if (StreamInfo::isStreamIdleTimeout(info)) {
     error_->error_code_ = ENVOY_REQUEST_TIMEOUT;
   } else {
     error_->error_code_ = ENVOY_STREAM_RESET;
   }
 
-  error_msg_details.push_back(absl::StrCat("ERROR_CODE: ", error_->error_code_));
+  error_msg_details.push_back(absl::StrCat("envoy_ec: ", error_->error_code_));
   std::vector<std::string> response_flags(info.responseFlags().size());
   std::transform(info.responseFlags().begin(), info.responseFlags().end(), response_flags.begin(),
                  [](StreamInfo::ResponseFlag flag) { return std::to_string(flag.value()); });
-  error_msg_details.push_back(absl::StrCat("RESPONSE_FLAGS: ", absl::StrJoin(response_flags, ",")));
+  error_msg_details.push_back(absl::StrCat("rsp_flags: ", absl::StrJoin(response_flags, ",")));
   if (info.protocol().has_value()) {
     // https://github.com/envoyproxy/envoy/blob/fbce85914421145b5ae3210c9313eced63e535b0/envoy/http/protocol.h#L13
-    error_msg_details.push_back(absl::StrCat("PROTOCOL: ", *info.protocol()));
+    error_msg_details.push_back(absl::StrCat("protocol: ", *info.protocol()));
```

Comment:
we could make this `http`, specifying which http version we're using, saves 4 chars, but `protocol` is a bit clearer.